### PR TITLE
Remove AlphanumComparator

### PR DESCRIPTION
**Pull request by RoiEXLab**
This class was only used twice.
Expected Difference (sorting):
Pool:
```
abc100
abc50
abc2
```
Old behaviour:
```
abc2
abc50
abc100
```
New behaviour:
```
abc100
abc2
abc50
```
(This replaces alphanumeric sorting with lexicographic sorting)

### CODE REVIEW

File: src/main/java/tools/map/making/ConnectionFinder.java [hunk 1]
```diff
@@ -137,9 +136,9 @@ public static void main(final String[] args) {
     // sort so that they are in alphabetic order (makes xml's prettier and easier to update in future)
```
Comment (ssoloff) [inline src/main/java/tools/map/making/ConnectionFinder.java]: You probably want to update this comment if you remove the alphanumeric sort.

File: src/main/java/tools/map/xml/creator/TerritoryConnectionsPanel.java [hunk 2]
```diff
@@ -113,9 +112,9 @@ private static void setTerritoryConnections(final Map<String, List<Area>> territ
     // sort so that they are in alphabetic order (makes xml's prettier and easier to update in future)
```
Comment (ssoloff) [inline src/main/java/tools/map/xml/creator/TerritoryConnectionsPanel.java]: You probably want to update this comment if you remove the alphanumeric sort.

**Comment by ssoloff** [comment]
Also note that, since Java 8, `Collections#sort()` simply delegates to `List#sort()`, so you may be able to get rid of the import for `Collections`, as well.  Although one could argue that `Collections.sort(myList)` reads better than `myList.sort(null)`.  Not sure why Oracle didn't add a zero-arg overload for `List#sort()`...

**Comment by ron-murhammer** [comment]
@RoiEXLab I agree with @ssoloff that I don't think functionally this change is a good idea. I'm pretty sure map makers would prefer the existing behavior.

**Comment by RoiEXLab** [comment]
No Problem, not really important anyway

**Pull request closed without merging**